packages/shared/src/hooks/useValueRef.ts

```diff
@@ -12,3 +13,19 @@ export function useValueRef<T>(ref: ValueRef<T>) {
 
     return useSubscription(subscription)
 }
+
+export function useValueRefDelayed<T>(ref: ValueRef<T>, latency = 500) {
```

Comment:
What is this used for?

Reply:
To delay UI updates and ensure synced those settings globally.